### PR TITLE
Update ncbi datasets downloader v.14.6.0 -> 14.13.2

### PR DESCRIPTION
**Pull request by gallardoalba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/ncbi_datasets/datasets_gene.xml [hunk 1]
```diff
@@ -352,7 +352,7 @@ dataformat
                 <assert_contents>
                     <has_text text="house mouse"/>
                     <has_text text="XR_004936704.1"/>
-                    <has_n_lines min="130"/>
```
Comment (gallardoalba) [inline tools/ncbi_datasets/datasets_gene.xml:355]: This min parameter seems to not work, so just replaced it by n, but I'm not sure if I'm missing something.

Comment (gallardoalba) [reply tools/ncbi_datasets/datasets_gene.xml:355]: I don't know, I just run it locally but the test failed; perhaps my Galaxy/planemo version is outdated. I'll reverse the changes, thanks!

File: tools/ncbi_datasets/datasets_gene.xml [hunk 2]
```diff
@@ -288,7 +288,7 @@ dataformat
                 <assert_contents>
                     <has_text text="baboon"/>
                     <has_text text="glyceraldehyde-3-phosphate dehydrogenase"/>
-                    <has_n_lines min="30"/>
+                    <has_n_lines n="456"/>
```
Comment (gallardoalba) [inline tools/ncbi_datasets/datasets_gene.xml:291]:
```suggestion
                    <has_n_lines min="30"/>
```

File: tools/ncbi_datasets/datasets_gene.xml [hunk 3]
```diff
@@ -352,7 +352,7 @@ dataformat
                 <assert_contents>
                     <has_text text="house mouse"/>
                     <has_text text="XR_004936704.1"/>
-                    <has_n_lines min="130"/>
+                    <has_n_lines n="137"/>
```
Comment (gallardoalba) [inline tools/ncbi_datasets/datasets_gene.xml:355]:
```suggestion
                    <has_n_lines min="130"/>
```

File: tools/ncbi_datasets/datasets_gene.xml [hunk 4]
```diff
@@ -386,7 +386,7 @@ dataformat
                 <assert_contents>
                     <has_text text="rat"/>
                     <has_text text="Brca1"/>
-                    <has_n_lines min="30"/>
+                    <has_n_lines n="331"/>
```
Comment (gallardoalba) [inline tools/ncbi_datasets/datasets_gene.xml:389]:
```suggestion
                    <has_n_lines min="30"/>
```

File: tools/ncbi_datasets/datasets_gene.xml [hunk 5]
```diff
@@ -432,7 +432,7 @@ dataformat
             <output name="gene_data_report">
                 <assert_contents>
                     <has_text text="human"/>
-                    <has_n_lines min="800"/>
+                    <has_n_lines n="836"/>
```
Comment (gallardoalba) [inline tools/ncbi_datasets/datasets_gene.xml:435]:
```suggestion
                    <has_n_lines min="800"/>
```

File: tools/ncbi_datasets/datasets_gene.xml [hunk 6]
```diff
@@ -490,7 +490,7 @@ dataformat
             <output name="gene_data_report">
                 <assert_contents>
                     <has_text text="human"/>
-                    <has_n_lines min="72000"/>
+                    <has_n_lines n="72612"/>
```
Comment (gallardoalba) [inline tools/ncbi_datasets/datasets_gene.xml:493]:
```suggestion
                    <has_n_lines min="72000"/>
```

File: tools/ncbi_datasets/datasets_gene.xml [hunk 7]
```diff
@@ -515,7 +515,7 @@ dataformat
...
```
Comment (gallardoalba) [inline tools/ncbi_datasets/datasets_gene.xml:518]:
```suggestion
                    <has_n_lines min="72000"/>
```

File: tools/ncbi_datasets/datasets_genome.xml [hunk 8]
```diff
@@ -199,7 +199,7 @@ $filters.exclude_atypical
             <output_collection name="genomic_gff" type="list">
                 <element name="GCF_000306695.2">
                     <assert_contents>
-                        <has_n_lines min="1000000"/>
+                        <has_n_lines n="1336377"/>
```
Comment (gallardoalba) [inline tools/ncbi_datasets/datasets_genome.xml:202]:
```suggestion
                        <has_n_lines min="1000000"/>
```

File: tools/ncbi_datasets/datasets_gene.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/ncbi_datasets/datasets_gene.xml:355]: Yes, you will need the latest version.

**Review by bernt-matthias — COMMENTED** [review]
Could you double check if there are any new command line arguments that are not covered yet. 

I guess that has not been done in https://github.com/galaxyproject/tools-iuc/pull/4978

Could you do me a favor and add me here https://github.com/galaxyproject/tools-iuc/blob/main/.github/CODEOWNERS for this tool?

**Comment by gallardoalba** [comment]
> Could you double check if there are any new command line arguments that are not covered yet.
> 
> I guess that has not been done in #4978
> 
> Could you do me a favor and add me here https://github.com/galaxyproject/tools-iuc/blob/main/.github/CODEOWNERS for this tool?

Sure!

**Comment by gallardoalba** [comment]
Is the planemo bot working @bernt-matthias?

**Comment by bernt-matthias** [comment]
> Is the planemo bot working

The update bot?

**Comment by gallardoalba** [comment]
> > Is the planemo bot working
> 
> The update bot?

Yes, I didn't see so much activity from it.

**Comment by nsoranzo** [comment]
I think something's wrong with it: https://github.com/planemo-autoupdate/autoupdate/actions/runs/3929849614/jobs/6798763845

**Comment by bernt-matthias** [comment]
Guess someone has to create `main` on https://github.com/planemo-autoupdate/tools-iuc/ ?

**Comment by nsoranzo** [comment]
> Guess someone has to create `main` on https://github.com/planemo-autoupdate/tools-iuc/ ?

Yep, just posted a message on the IUC Gitter channel.

**Comment by nsoranzo** [comment]
> > Guess someone has to create `main` on https://github.com/planemo-autoupdate/tools-iuc/ ?

That's been done. Hopefully https://github.com/planemo-autoupdate/autoupdate/commit/83df926526c3ab4a0a438ca2a8d5c5fc077b8376 was the last fix needed, I see new PRs coming in.

**Review by bgruening — APPROVED** [review]

**Comment by bernt-matthias** [comment]
FYI https://github.com/galaxyproject/tools-iuc/issues/4994

**Comment by gallardoalba** [comment]
> Could you double check if there are any new command line arguments that are not covered yet.
> 
> I guess that has not been done in #4978
> 
> Could you do me a favor and add me here https://github.com/galaxyproject/tools-iuc/blob/main/.github/CODEOWNERS for this tool?

I checked, and there's not changes in the command lines. Also I included you in the CODEOWNERS file.

**Comment by gallardoalba** [comment]
Orthologs service currently not working: https://www.ncbi.nlm.nih.gov/gene/672/ortholog/similargenes/

**Pull request closed without merging**